FastenersCmd.py

```diff
@@ -715,5 +728,6 @@ def onDocumentRestored(self, obj):
 FastenerBase.FSAddFastenerType("Stud")
 FastenerBase.FSAddFastenerType("HeatSet", False)
 FastenerBase.FSAddFastenerType("RetainingRing", False)
+FastenerBase.FSAddFastenerType("T-Slot", False)
```

Comment:
Is really needed this line, what changes?
I think it should be "TSlot" instead of "T-Slot".

Reply:
the types are used for example in fschangeparams.py:
![image](https://github.com/shaise/FreeCAD_FastenersWB/assets/5813568/d8b7e43a-f758-4dd0-8b63-515521530cc1)